02.calendar/calendar.js

```diff
@@ -0,0 +1,42 @@
+function print_calendar(month, year){
+    print_header(month, year);
+    const first_day_of_month = new Date(year, month -1, 1);
+    const first_wday = first_day_of_month.getDay();
+    const last_day_of_month = new Date(year, month, 0);
+    const dates = getDates(first_day_of_month, last_day_of_month);
+    let str =  '   '.repeat(first_wday) + (first_wday>2?' '.repeat(first_wday-2):'')
```

Comment:
```suggestion
    let str =  '   '.repeat(first_wday) + (first_wday > 2 ? ' '.repeat(first_wday - 2) : '')
```

のようにスペースを入れましょう。

https://beautifier.io/ のようなformatterも試してみてください。